### DISCUSSION TITLE
Can't select a valid directory for comparison

### DISCUSSION THREAD

**Discussion by Mailleweaver** [WinMerge/winmerge#1492] [Q&A] [answered]
I'm trying to compare a directory in my computer with a directory in my connected android phone. WinMerge will not allow me to select the phone directory using the 'browse' feature, and if I manually copy the path from Windows Explorer then WinMerge reports that the path is invalid. How do I select this directory for comparison?

Here's a snip of the 'open' dialogue showing the error message while the directory in question is successfully open with Windows Explorer in the background.
![WinMerge](https://user-images.githubusercontent.com/1341522/191601544-5555a659-3b0e-4d01-91f6-4982559fd27d.png)

**Comment by sdottaka** [comment, marked as answer]
Unfortunately WinMerge does not support devices that communicate via MTP/PTP, such as smartphones.

Beyond Compare seems to support MTP devices.

**Reply by Mailleweaver** [reply]
I guess I have no use for this program, then. Thanks.
